pkg/kubectl/cmd/cp.go

```diff
@@ -124,6 +119,14 @@ func runCopy(f cmdutil.Factory, cmd *cobra.Command, out, cmderr io.Writer, args
 	if err != nil {
 		return err
 	}
+
+	if len(srcSpec.PodName) != 0 && len(destSpec.PodName) != 0 {
+		if _, err := os.Stat(args[0]); err == nil {
+			return copyToPod(f, cmd, out, cmderr, fileSpec{File: args[0]}, destSpec)
+		}
+		return cmdutil.UsageErrorf(cmd, "both src and dest look like remote file specification and src doesn't exist in local filesystem")
```

Comment:
This error is a bit confusing. They both look like remote file specifications because we are now allowing them to have filenames with `:`. Maybe the error should just mention the file not existing